ci/templates/build_docker.yml

```diff
@@ -48,7 +48,13 @@ steps:
     # Create a temporary container from the runtime image but don't start it -- it's just
     # used to extract the packaged component for archiving and later release.
     container_id=$(docker create ${runtime_image_for_build_id})
-    docker cp $container_id:${{ parameters.artifactPath }} "$(ARTIFACT_DIRECTORY)/"
+    
+    artifacts="${{ parameters.artifactPaths }}"
```

Comment:
Do you need this separate variable or can you just do

```
for artifact in ${{ parameters.artifactPaths }}; do
```

Reply:
I've actually played around with that and wasn't able to get it working with above, or by quoting it.  So I'm back with the intermediate variable.